diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -123,6 +123,23 @@ namespace diff_drive_controller
       wheel_speed_limiter_ = wheel_speed_limiter;
     }
 
+  protected:
+    /*
+     * \brief Default wheel speed limiter function
+     * \param [in, out] left_velocity  Left wheel velocity (not used)
+     * \param [in, out] right_velocity Right wheel velocity (not used)
+     */
+    virtual void wheelSpeedLimiter(double& left, double& right)
+    {
+      if (wheel_speed_limiter_)
```

Comment:
I'd rather go this route for now as it doesn't break r_v_i. I expect we'll change that eventually and then we can indeed remove this, but I don't want to have to rush into it.